appendices/pre-review-package-requirements.md

```diff
@@ -0,0 +1,12 @@
+
+- [ ] **Fit**: The package meets criteria for [fit](https://www.pyopensci.org/peer-review-guide/about-peer-review/aims-and-scope.html#package-scope) and [overlap](https://www.pyopensci.org/peer-review-guide/about-peer-review/aims-and-scope.html#package-overlap).
+    - [ ] **Package overlap:** That package doesn't fully overlap with functionality of other packages that have already been submitted to pyOpenSci
+- [ ] **Documentation** The package has sufficient documentation available online (README, sphinx docs) to allow us to evaluate package function and scope *without installing the package*. This includes:
+  - [ ] Get started vignettes or tutorials that help a user understand how to use the package and what it can do for them
+  - [ ] API documentation - this includes clearly written doc strings with variables defined using a standard docstring format
+- [ ] **README.md File:** The package has a README.md file with clear explanation of what the packages does and instructions on how to install it along with development instructions.  
+   - [ ] the README file clearly articulates the function of the package
+- [ ] **CONTRIBUTING.md file:** The package has a `CONTRIBUTING.md` file that details how to install and contribute to the package. 
+- [ ] **Automated tests:** Package has a testing suite and is tested via GitHub actions or another Continuous Integration service.
+- [ ] **License:** The package has an OSI accepted license
+- [ ] **Repository:** The package repository link resolves correctly
```

Comment:
```suggestion
    - [ ] **Package overlap** That package doesn't entirely overlap with the functionality of other packages that have already been submitted to pyOpenSci.
- [ ] **Documentation** The package has sufficient online documentation (README, sphinx docs) to allow us to evaluate package function and scope *without installing the package*. This includes:
  - [ ] Get started vignettes or tutorials that help users understand how to use the package and what it can do for them.
  - [ ] API documentation - this includes clearly written doc strings with variables defined using a standard docstring format.
- [ ] **README.md File** The package has a README.md file with a clear explanation of what the package does and instructions on how to install it, along with development instructions.  
   - [ ] README file clearly articulates the function of the package.
- [ ] **CONTRIBUTING.md file** The package has a `CONTRIBUTING.md` file that details how to install and contribute to the package. 
- [ ] **Automated tests** Package has a testing suite and is tested via GitHub actions or another Continuous Integration service.
- [ ] **License** The package has an OSI accepted license.
- [ ] **Repository** The package repository link resolves correctly.
```

Reply:
Wasn't this the same one as before? 🤔 Shouldn't it be the same text?

Reply:
sorry @arianesasso we could have some duplicate text. i pulled out things that I thought should be include files to avoid duplicated but slightly different text. but it is totally possible that i made a mistake! are you referring to another template here where you saw the same text?  many thanks!